src/_tests_/editTodoList.js

```diff
@@ -0,0 +1,53 @@
+import localStorageMock from '../_mocks_/localStorage.js';
+
+const { checkCompleted } = require('../_mocks_/editTodoList.js');
+
+describe(' checking status completed localstorage', () => {
+  document.body.innerHTML = `
+          <li class="list-container d-flex">
+            <input type="checkbox" class="checkbox" checked>
+            <p class="items show"   contenteditable="true">Hello Todo's</p>
+            <i class="fas fa-ellipsis-v  dots"></i>
+            <i class="far fa-trash-alt show delete"></i>
+          </li>
+          <button class="btn">Clear all Completed</button>
+          `;
+  test('function to check the complete statues false', () => {
+    const list = localStorageMock;
+    // addTodoToList('take a break', false, 0, list);
+    const task = document.querySelector('.items').innerText;
+    const element = document.querySelector('.checkbox');
+    const bool = false;
+    checkCompleted(list, task, element, bool);
+    const jsonId = '0';
```

Comment:
- [ ] good job using the `describe` block 👍 as you are testing in this file the file name format should look like `[...].test.js`